### PR TITLE
Balance: Absorb DNA cooldown

### PR DESCRIPTION
**Pull request by Kaedwuff**
On request from a staff member, I have implemented a delay on using Absorb DNA.  Changelings will be unable to use the ability for a full minute after successfully completing it.

### CODE REVIEW

File: code/game/gamemodes/changeling/changeling_powers.dm [hunk 1]
```diff
@@ -143,6 +144,10 @@ var/global/list/possible_changeling_IDs = list("Alpha","Beta","Gamma","Delta","E
 	var/datum/changeling/changeling = changeling_power(0,0,100)
 	if(!changeling)	return
 
+	if (changeling.justate + 60 SECONDS > world.time)
```
Comment (Sindorman) [inline code/game/gamemodes/changeling/changeling_powers.dm]: timer might be a better way

**Comment by Kaedwuff** [comment]
!review

**Comment by Kaedwuff** [comment]
!review

File: code/game/gamemodes/changeling/changeling_powers.dm (hunk 1 above)
Comment (Lohikar) [reply code/game/gamemodes/changeling/changeling_powers.dm]: A timer is more complex and more overhead for 0 benefit. It's just a cooldown, it doesn't need to do something when it clears.

**Comment by Kaedwuff** [comment]
!review

**Pull request merged**